src/GlobalOptions.cmake

```diff
@@ -1,27 +0,0 @@
-set(args
```

Comment:
Now that this one is removed, we should update the last readme example accordignly.

Reply:
resolved in a7eb218